docs/apps/https.md

```diff
@@ -0,0 +1,61 @@
+# HTTPS App
+
+This app allows you to register a new asset into Badrap simply by fetching a URL. The IP address from which the request originates will be added as a new asset under your user account. You will receive security notifications about the asset you have registered, and you can use other Badrap apps to enrich or to interact with the asset. 
+
+## Install the app
+
+1. Open the [HTTPS app page](https://badrap.io/apps/https). 
+
+<div style="text-align: center;">
+   <img src="./https-10-install.png" style="max-width: 95%; width: 480px;" />
+</div>
+
+2. Click **Install**. The app will ask for your permission to create new assets under your Badrap account. 
+
+<div style="text-align: center;">
+   <img src="./https-15-perms.png" style="max-width: 95%; width: 480px;" />
+</div>
+
+3. Click on **Install the app**. The app is now installed. 
+
+## Add a new asset
+
+1. Go to the [HTTPS app page](https://badrap.io/apps/https). Click on **Generate new address**. 
+
+<div style="text-align: center;">
+   <img src="./https-20-generate-new-url.png" style="max-width: 95%; width: 480px;" />
+</div>
+
+2. Click on the **Copy to clipboard** button to copy the URL onto your clipboard. 
+
+3. On the host that you want to add as a new asset into Badrap, fetch the URL using a suitable application. For command-line usage, common tools for fetching an URL are e.g. [Wget](https://www.gnu.org/software/wget/) or [curl](https://curl.se/). These command-line tools are commonly available for Linux distributions. curl is built into Mac OS X, and into Windows 10 command line and PowerShell environments. For desktop hosts, you can simply fetch the URL with any web browser. 
+
+Example using wget:
+```
+wget https://http-app.badrap.io/app/claim/1abc...
+```
+
+Example using curl:
+```
+curl https://http-app.badrap.io/app/claim/1abc...
+```
+
```

Comment:
Currently the wget and curl example paragraphs are not indented to the same level as the rest of the step:

<img width="795" alt="Screenshot 2021-05-14 at 11 45 05" src="https://user-images.githubusercontent.com/19776768/118245859-f488a280-b4a9-11eb-9a64-3ca5651c8d9c.png">

Could this be fixed?

Reply:
Fixed